ocaml/xapi/xapi_pgpu.ml

```diff
@@ -312,12 +312,7 @@ let get_remaining_capacity ~__context ~self ~vgpu_type =
 
 let assert_can_run_VGPU ~__context ~self ~vgpu =
```

Comment:
I can't seem to find any callers to this function apart from in the unit tests. Am I missing something?

Reply:
I think that it was removed in 30d9517cd27450ca06beff0b12f44e966c030703 and replaced by the dry-run stuff in `allocate_vgpu_to_gpu`?